models/flower.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class Flower(models.Model):
+    _name = 'flower'
+    _description = 'Flower'
+
+    name = fields.Char(required=True)
+    scientific_name = fields.Char()
```

Comment:
It is always good to set a label for your fields using the **string** attribute on the fields classes. If you do not, Odoo will generate one by default based on the field name. In this case scientific_name will be converted into a label by capitalizing the words and replacing underscores by spaces.